app/bundles/CampaignBundle/Model/EventModel.php

```diff
@@ -845,6 +845,7 @@ public function triggerScheduledEvents(
 
             // Get a count
             $events = $repo->getScheduledEvents($campaignId, false, $limit);
+            $this->logger->debug('CAMPAIGN: events #'.print_r($events, true));
```

Comment:
@kuzmany do you think you could remove this line? It may cause the log sizes to explode.

Reply:
Removed